code/game/objects/items/weapons/material/kitchen.dm

```diff
@@ -94,7 +94,7 @@
 
 // Identical to the tactical knife but nowhere near as stabby.
 // Kind of like the toy esword compared to the real thing.
-/obj/item/weapon/material/kitchen/utensil/knife/boot
+/obj/item/weapon/material/kitchen/utensil/knife_bowie/boot
```

Comment:
this is not how subtypes work; the new types should be descended from `/obj/item/weapon/material/kitchen/utensil/knife/boot`

Reply:
no it needs to if els it will not work on venders